### PR TITLE
make the title of a competency more precise

### PR DESCRIPTION
**Pull request by CaptainSifff**
I changed the title to "use software repositories" to delineate them from the domain repos in the research part.

### CODE REVIEW

**Review by jpthiele — APPROVED** [review]

**Review by MakisH — APPROVED** [review]
That's a helpful distinction! :+1:

**Comment by mhagdorn** [comment]
I think we also need to add something to the DOMREP section. Maybe change the sentence to 
```
The RSE needs to be aware of any domain specific repositories which may contain in addition 
to software also data sets and catalogues.
```

**Review by mhagdorn — APPROVED** [review]

File: competencies.md [hunk 1]
```diff
@@ -462,6 +462,8 @@ systems.
 
 The RSE should be able to identify and use fitting public platforms (so-called software repositories or "repos")
 to share the artefacts they have created and invite the public to scrutinise them in public reviews.
+These software repositories usually provide facilities for software development, which 
+differentiate them from the later to be described domain repositories.
```
Comment (tobyhodges) [inline competencies.md:466]:
```suggestion
differentiate them from the domain repositories described later.
```

**Review by tobyhodges — APPROVED** [review]
Suggesting one minor polishing change, but otherwise this looks good

**Review by jngrad — APPROVED** [review]

**Pull request merged**